README.md

```diff
@@ -66,33 +66,26 @@ _ = json(.get, stringURL)
     .observeOn(MainScheduler.instance)
     .subscribe { print($0) }
 
+// validation
 _ = request(.get, stringURL)
-    .flatMap { request in
-        return request.validate(statusCode: 200..<300)
-        .validate(contentType: ["text/json"])
-            .rx.json()
-    }
+    .validate(statusCode: 200..<300)
+    .validate(contentType: ["application/json"])
+    .responseJSON()
     .observeOn(MainScheduler.instance)
     .subscribe { print($0) }
 
 // progress
 _ = request(.get, stringURL)
     .flatMap {
-        $0
-        .validate(statusCode: 200 ..< 300)
-        .validate(contentType: ["text/json"])
-        .rx.progress()
+        $0.rx.progress()
     }
     .observeOn(MainScheduler.instance)
     .subscribe { print($0) }
 
 // just fire upload and display progress
 _ = upload(Data(), urlRequest: try! RxAlamofire.urlRequest(.get, stringURL))
     .flatMap {
-        $0
-        .validate(statusCode: 200 ..< 300)
-            .validate(contentType: ["text/json"])
-            .rx.progress()
+        $0.rx.progress()
```

Comment:
Was there any need to change this as part of this PR?

Reply:
I wanted the example to focus on `progress` by not "polluting" with validation. So I made it more simple and focused. I can revert this if do not agree with me. 😉